src/Scenario/StatusWatcherScenario.php

```diff
@@ -21,9 +21,9 @@
  *
  * Uses two telegram, connections (infoClient, monitoringClient).
  *
- * Requires files: `first.authkey`, `second.authkey`.
+ * Requires files: `first.authkey`.
  */
-class MyTgClientDebug implements StatusWatcherCallbacks, ClientDebugLogger, ScenarioInterface
+class StatusWatcherScenario implements StatusWatcherCallbacks, ClientDebugLogger, ScenarioInterface
```

Comment:
This client could be redundant, possibly could be removed

Reply:
let's leave it for now, we have scenario for status watching

Reply:
Lets get rid of info client then - it is redundant

Reply:
done